letsencrypt-apache/letsencrypt_apache/configurator.py

```diff
@@ -859,6 +863,10 @@ def _add_dummy_ssl_directives(self, vh_path):
                             "insert_key_file_path")
         self.parser.add_dir(vh_path, "Include", self.mod_ssl_conf)
 
+    def _add_servername(self, servername, vh_path):
+        self.parser.add_dir(vh_path, "ServerName", servername)
```

Comment:
I haven't followed this whole patch yet, but instinctively what I'm expecting around here is "if the vhost doesn't have a ServerName, add one with the name `example.com`; if it already has a ServerName, add a ServerAlias with the name `example.com`".